src/ontology/config/properties.txt

```diff
@@ -20,6 +20,8 @@ http://www.w3.org/2004/02/skos/core#narrowMatch
 http://www.w3.org/2004/02/skos/core#relatedMatch
 http://www.w3.org/2004/02/skos/core#exactMatch
 http://www.w3.org/2004/02/skos/core#closeMatch
+rdfs:comment
 rdfs:label
 rdfs:seeAlso
 owl:deprecated
+http://purl.org/dc/terms/description
```

Comment:
## `xmlns:terms="http://purl.org/dc/terms/"`
- [ ] Advise @matentzn or @twhetzel

I originally added an entry `dcterms:description` to `config/properties.txt`, but it was not a recognized namespace, so when I converted `longDefinition` to this, it got removed. I changed to `http://purl.org/dc/terms/description`, but I got an unexpected namespace `terms` added.

I actually had the same thing happen for an OMOP ingest I was doing for TIMS. I was doing `robot template ... --prefix "OMOP: https://athena.ohdsi.org/search-terms/terms/"`, and it ignored my namespace and did `xmnls:terms` instead.

Does this bother you at all or cause any problems for us? Should I keep it as is? Make an issue to handle this later? Or handle this before merging this PR?

Any ideas what might be causing the issue?
I know that we just had the same problem with it creating `xmlns:schema="http://id.who.int/icd/schema/"`. When that happened, you asked me to check the original `.owl` that we got from WHO and see if that namespace was already there, and indeed it was. However, I just checked it and this 'terms' namespace is not there.

Reply:
If it's just namespace declarations don't worry about it. In RDF they are purely cosmetic. Now in semsql all of this may matter more; but then you simple extend the prefixmap for that correctly. So yes, you can leave it as is.